src/sql/magic.py

```diff
@@ -180,6 +181,30 @@ def _mutex_autopandas_autopolars(self, change):
                 setattr(self, other, False)
                 print(f"Disabled '{other}' since '{change['name']}' was enabled.")
 
+    def check_random_arguments(self, line="", cell=""):
+        # check only for cell magic
+        if cell != "":
+            tokens = line.split()
+            arguments = []
+
+            # Iterate through the tokens to separate arguments and SQL code
+            # If the token starts with "--", it is an argument
+            breakLoop = False
+            for token in tokens:
+                if token.startswith("--"):
```

Comment:
there are short options as well, so we also need to check if the options start with a single dash

Reply:
update it now